### DISCUSSION TITLE
How to restore multi-step zoom functionality in Fullscreen mode?

### DISCUSSION THREAD

**Discussion by dineshkumarrajaguru** [igordanchenko/yet-another-react-lightbox#211] [Q&A] [answered]
Team,

I'm utilizing the Inline plugin and successfully implemented the opening of a Lightbox popup upon clicking any slide using the Fullscreen plugin. However, I'm encountering issues with the Zoom ratio, as there is only one level of zoom in full-screen mode.

Is there a solution to restore multi-step zoom functionality over slides displayed in Fullscreen mode?

**Comment by igordanchenko** [comment, marked as answer]
Can you see if increasing `maxZoomPixelRatio` value helps? 

https://yet-another-react-lightbox.com/plugins/zoom#Documentation

**Reply by dineshkumarrajaguru** [reply]
Thanks @igordanchenko I got it working by increasing the maxZoomPixelRatio value.
